integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -892,7 +892,7 @@ void testK8SEventsStartStopWatchingNSWithRegExp(boolean enableClusterRoleBinding
     upgradeAndVerifyOperator(opNamespace, opParams);
 
     logger.info("verify NamespaceWatchingStopped event is logged in namespace {0}", domainNamespace5);
-    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace5, null, "Normal", timestamp,
+    checkNamespaceWatchingStoppedEvent(opNamespace, domainNamespace5, domainUid, "Normal", timestamp,
```

Comment:
domainUid should be null as before. NamespaceWatchingStopped event will not contin domainUID